types/sass/index.d.ts

```diff
@@ -206,3 +207,63 @@ export interface Result {
 
 export function render(options: Options, callback: (exception: SassException, result: Result) => void): void;
 export function renderSync(options: Options): Result;
+export namespace types {
```

Comment:
Can you use a indentation of 4 here, like the rest above 😃

Reply:
And maybe switch the namespace to `sass` instead of `types`. That way it is a lot more specific.

Reply:
addressed